README.md

```diff
@@ -226,6 +226,13 @@ ALTER TABLE users
 ADD COLUMN created_at timestamp GENERATED ALWAYS AS (id::timestamp) STORED;
 ```
 
+Cast timestamp to [ulid][]:
+
+```sql
+-- gets all users where the ID was created on 2023-09-15, without using another column and taking advantage of the index
+SELECT * FROM users WHERE id BETWEEN timestamp_to_ulid('2023-09-15') AND '2023-09-16'::timestamp::ulid;
```

Comment:
```suggestion
SELECT * FROM users WHERE id BETWEEN '2023-09-15'::timestamp::ulid AND '2023-09-16'::timestamp::ulid;
```

Reply:
I left like this to showcase that the function can be called directly too. Do you think is good to only show the cast ?

Reply:
Yeah, I prefer casts to be recommended.

Reply:
Done!